src/index.js

```diff
@@ -24,10 +24,12 @@ export default function (babel) {
           return
         }
         const {id: {name}} = node
-        const {referencePaths} = path.scope.getBinding(name)
-        referencePaths.forEach(reference => {
-          identifiers.add(reference)
-        })
+        if(path.scope.getBinding(name)){
```

Comment:
Sweet! Everything still works. I'll go ahead and merge/publish. We should be good to go now.